From b16904e77b546c813b0cea1296d679b56da4471b Mon Sep 17 00:00:00 2001
From: Billy Laws <blaws05@gmail.com>
Date: Thu, 26 Dec 2019 13:57:44 +0100
Subject: [PATCH] HACK: use platform sig for shieldtech

---
 shield/shieldtech/Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/shield/shieldtech/Android.mk b/shield/shieldtech/Android.mk
index 5dcf672..b607400 100644
--- a/shield/shieldtech/Android.mk
+++ b/shield/shieldtech/Android.mk
@@ -18,7 +18,7 @@ include $(CLEAR_VARS)
 LOCAL_MODULE               := NvShieldTech
 LOCAL_MODULE_TAGS          := optional
 LOCAL_SRC_FILES            := app/NvShieldTech.apk
-LOCAL_CERTIFICATE          := PRESIGNED
+LOCAL_CERTIFICATE          := platform
 LOCAL_MODULE_CLASS         := APPS
 LOCAL_MODULE_SUFFIX        := $(COMMON_ANDROID_PACKAGE_SUFFIX)
 LOCAL_PRIVILEGED_MODULE    := true
-- 
2.24.0

