From 92348d822c2d167ae546599fe1f2341243f988ff Mon Sep 17 00:00:00 2001
From: Billy Laws <blaws05@gmail.com>
Date: Sat, 8 Jan 2022 15:01:09 +0000
Subject: [PATCH] uevent: Add upstream CEC perms

---
 initfiles/ueventd.t210ref.rc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/initfiles/ueventd.t210ref.rc b/initfiles/ueventd.t210ref.rc
index fec2879..01b032c 100644
--- a/initfiles/ueventd.t210ref.rc
+++ b/initfiles/ueventd.t210ref.rc
@@ -86,6 +86,7 @@
 /dev/ttyTHS3	0660	bluetooth	bluetooth
 /sys/devices/platform/tegra_cec cec_logical_addr_config	0660	system	system
 /dev/tegra_cec	0660	system	system
+/dev/cec0	0660	system	system
 # pmqos
 /dev/constraint_cpu_freq 0660 system system
 /dev/constraint_gpu_freq 0660 system system
-- 
2.34.1

