diff --git a/service/java/com/android/server/wifi/WifiStateMachinePrime.java b/service/java/com/android/server/wifi/WifiStateMachinePrime.java
index 85ea001..f1314a9 100644
--- a/service/java/com/android/server/wifi/WifiStateMachinePrime.java
+++ b/service/java/com/android/server/wifi/WifiStateMachinePrime.java
@@ -411,10 +411,10 @@ public class WifiStateMachinePrime {
                             Log.d(TAG, "Client mode state change from previous manager");
                             return HANDLED;
                         }
-                        Log.d(TAG, "ClientMode failed, return to WifiDisabledState.");
+                        Log.d(TAG, "ClientMode failed, return to WifiScanOnlyMode.");
                         // notify WifiController that ClientMode failed
-                        mClientModeCallback.onStateChanged(WifiManager.WIFI_STATE_UNKNOWN);
-                        mModeStateMachine.transitionTo(mWifiDisabledState);
+                        mClientModeCallback.onStateChanged(WifiManager.WIFI_STATE_ENABLED);
+                        mModeStateMachine.transitionTo(mScanOnlyModeActiveState);
                         break;
                     case CMD_CLIENT_MODE_STOPPED:
                         if (mListener != message.obj) {
@@ -496,10 +496,9 @@ public class WifiStateMachinePrime {
                             return HANDLED;
                         }
 
-                        Log.d(TAG, "ScanOnlyMode failed, return to WifiDisabledState.");
-                        // notify WifiController that ScanOnlyMode failed
-                        mScanOnlyCallback.onStateChanged(WifiManager.WIFI_STATE_UNKNOWN);
-                        mModeStateMachine.transitionTo(mWifiDisabledState);
+                        Log.d(TAG, "ScanOnlyMode failed, retry.");
+                        mClientModeCallback.onStateChanged(WifiManager.WIFI_STATE_ENABLED);
+                        mModeStateMachine.transitionTo(mScanOnlyModeActiveState);
                         break;
                     case CMD_SCAN_ONLY_MODE_STOPPED:
                         if (mListener != message.obj) {
