diff --git a/bluetooth/1.0/default/h4_protocol.cc b/bluetooth/1.0/default/h4_protocol.cc
index df40507..bc084c0 100644
--- a/bluetooth/1.0/default/h4_protocol.cc
+++ b/bluetooth/1.0/default/h4_protocol.cc
@@ -58,7 +58,7 @@ void H4Protocol::OnPacketReady() {
       sco_cb_(hci_packetizer_.GetPacket());
       break;
     default:
-      LOG_ALWAYS_FATAL("%s: Unimplemented packet type %d", __func__,
+      ALOGE("%s: Unimplemented packet type %d", __func__,
                        static_cast<int>(hci_packet_type_));
   }
   // Get ready for the next type byte.
@@ -87,7 +87,7 @@ void H4Protocol::OnDataReady(int fd) {
     if (hci_packet_type_ != HCI_PACKET_TYPE_ACL_DATA &&
         hci_packet_type_ != HCI_PACKET_TYPE_SCO_DATA &&
         hci_packet_type_ != HCI_PACKET_TYPE_EVENT) {
-      LOG_ALWAYS_FATAL("%s: Unimplemented packet type %d", __func__,
+      ALOGE("%s: Unimplemented packet type %d", __func__,
                        static_cast<int>(hci_packet_type_));
     }
   } else {
diff --git a/bluetooth/1.0/default/mct_protocol.cc b/bluetooth/1.0/default/mct_protocol.cc
index 2a59187..f0b98f0 100644
--- a/bluetooth/1.0/default/mct_protocol.cc
+++ b/bluetooth/1.0/default/mct_protocol.cc
@@ -44,7 +44,7 @@ size_t MctProtocol::Send(uint8_t type, const uint8_t* data, size_t length) {
     return WriteSafely(uart_fds_[CH_CMD], data, length);
   if (type == HCI_PACKET_TYPE_ACL_DATA)
     return WriteSafely(uart_fds_[CH_ACL_OUT], data, length);
-  LOG_ALWAYS_FATAL("%s: Unimplemented packet type = %d", __func__, type);
+  ALOGE("%s: Unimplemented packet type = %d", __func__, type);
   return 0;
 }
 

