diff --git a/common/nvgpu/Android.mk b/common/nvgpu/Android.mk
index bd5896f..95475f9 100644
--- a/common/nvgpu/Android.mk
+++ b/common/nvgpu/Android.mk
@@ -380,18 +380,6 @@ LOCAL_MODULE_OWNER         := nvidia
 LOCAL_VENDOR_MODULE        := true
 include $(BUILD_NVIDIA_COMMON_PREBUILT)
 
-include $(CLEAR_VARS)
-LOCAL_MODULE               := vendor.nvidia.hardware.graphics.display@1.0
-LOCAL_SRC_FILES_32         := lib/vendor.nvidia.hardware.graphics.display@1.0.so
-LOCAL_SRC_FILES_64         := lib64/vendor.nvidia.hardware.graphics.display@1.0.so
-LOCAL_MULTILIB             := both
-LOCAL_MODULE_SUFFIX        := .so
-LOCAL_MODULE_CLASS         := SHARED_LIBRARIES
-LOCAL_MODULE_TAGS          := optional
-LOCAL_MODULE_OWNER         := nvidia
-LOCAL_VENDOR_MODULE        := true
-include $(BUILD_NVIDIA_COMMON_PREBUILT)
-
 include $(CLEAR_VARS)
 LOCAL_MODULE               := vendor.nvidia.hardware.graphics.display@1.0-impl
 LOCAL_SRC_FILES_32         := lib/vendor.nvidia.hardware.graphics.display@1.0-impl.so
